src/main/scala/microsites/MicrositeKeys.scala

```diff
@@ -167,14 +201,64 @@ trait MicrositeAutoImportSettings extends MicrositeKeys {
 
   lazy val fr = new FileReader
 
+  def createVersionsJson(targetDir: String, content: List[Version]): File = {
+    val jekyllDir  = "jekyll"
+    val targetPath = s"$targetDir$jekyllDir/_data/versions.json"
+    createFile(targetPath)
+    writeContentToFile(content.asJson.toString, targetPath)
+    targetPath.toFile
+  }
+
+  def generateVersionList(versionStringList: List[String], ownVersion: String) = {
+    versionStringList
+      .map(version => {
+        Version(version, own = (ownVersion == version))
+      })
+      .toList
+  }
+
+  def pluginProjects(pluginName: String): Option[Array[String]] = {
+    val sbtPluginsOutput = "sbt --error plugins".lineStream
+    val pluginLine =
+      sbtPluginsOutput.find(line => line.trim.startsWith(s"$pluginName: enabled in "))
+
+    pluginLine match {
+      case Some(line) => Some(line.trim.stripPrefix(s"$pluginName: enabled in ").split(", "))
+      case None       => None
+    }
```

Comment:
```scala
pluginLine.map(_.trim.stripPrefix(s"$pluginName: enabled in ").split(", "))
```

?

Reply:
Would the use of `map` there return a `None` in the case that `pluginLine` was one?  I was under the assumption that `map` would only work with _right_ values, but I wasn't sure what would happen for `None` value.

Reply:
Addressed in https://github.com/47deg/sbt-microsites/pull/379/commits/41e996fd7cf2934d9ab7d07d58d722506ab70b9b